velox/connectors/CMakeLists.txt

```diff
@@ -15,6 +15,10 @@ add_library(velox_connector Connector.cpp)
 
 target_link_libraries(velox_connector velox_config velox_vector)
 
+add_library(velox_write_protocol WriteProtocol.cpp)
```

Comment:
Any particular reason not to add WriteProtocol.cpp to velox_connector?

Reply:
Revising. Adding it to velox_connector makes it easier.